compiler/ballerina-lang/src/main/resources/grammar/BallerinaLexer.g4

```diff
@@ -341,7 +341,7 @@ EscapeSequence
 
 fragment
 UnicodeEscape
-    :   '\\' 'u' HexDigit HexDigit HexDigit HexDigit
+    :   '\\' 'u' LEFT_BRACE HexDigit+ RIGHT_BRACE
```

Comment:
Adding braces around the value looks bit weird. Is there any specific reason to do this? 😅

Reply:
It seems like, In the JS world, it is a norm. :) Here is the Spec change related to this. https://github.com/ballerina-platform/ballerina-spec/commit/eb2fa56ac777605dee3a26b1b7c389c1e5f7601d